pages/expandable-section/permutations.page.tsx

```diff
@@ -124,6 +124,12 @@ const permutations = createPermutations<ExpandableSectionProps>([
     children: ['Sample content'],
     headingTagOverride: [undefined, 'h2', 'h3'],
   },
+  {
```

Comment:
The container variant is already covered further above in line 84. It is covered using a Header component whereas the default and footer variant are covered using plain text, because the other combinations look broken anyway.